python/lsst/pipe/supertask/activator.py

```diff
@@ -356,7 +362,7 @@ def load_super_task(super_taskname):
 
         if class_config_instance is None:
             # No ConfigClass associated to SuperTask was found
-            raise TaskNameError(
+            print('WARNING: ' +
                 ' No SuperConfig Class %s found: Task or similiar' % (super_taskname.replace('Task',
                                                                                              'Config')), None)
```

Comment:
This commit is dropped because it indeed should raise.  

`None` is wanted in TaskNameError currently.